Lib/_ios_support.py

```diff
@@ -0,0 +1,70 @@
+import sys
+try:
+    from ctypes import cdll, c_void_p, c_char_p, util
+except ImportError:
+    # ctypes is an optional module. If it's not present, we're limited in what
+    # we can tell about the system, but we don't want to prevent the platform
+    # module from working.
+    objc = None
+else:
+    # ctypes is available. Load the ObjC library, and wrap the objc_getClass,
+    # sel_registerName and objc_msgSend methods
+    objc = cdll.LoadLibrary(util.find_library("objc"))
+    if objc._name:
+        objc.objc_getClass.restype = c_void_p
+        objc.objc_getClass.argtypes = [c_char_p]
+        objc.sel_registerName.restype = c_void_p
+        objc.sel_registerName.argtypes = [c_char_p]
+        # The calls to objc_msgSend all have no arguments; but the return types
+        # vary with the specific call.
+        objc.objc_msgSend.argtypes = [c_void_p, c_void_p]
+    else:
+        # Failed to load the objc library
+        objc = None
+
+
+def get_platform_ios():
+    # Determine if this is a simulator using the multiarch value
+    is_simulator = sys.implementation._multiarch.endswith("simulator")
+
+    # We can't use ctypes; abort
+    if not objc:
+        return
```

Comment:
Explicit return:

```suggestion
        return None
```

(See "Be consistent in return statements" in [PEP 8](https://peps.python.org/pep-0008/#programming-recommendations).)